test/unittests/execute_call_test.cpp

```diff
@@ -615,13 +620,14 @@ TEST(execute_call, call_initial_depth)
     const auto wasm = from_hex("0061736d01000000010401600000020b01036d6f6403666f6f0000");
 
     auto module = parse(wasm);
-    auto host_foo = [](Instance& /*instance*/, const Value*, int depth) -> ExecutionResult {
+    constexpr auto host_foo = [](void*, Instance&, const Value*,
+                                  int depth) noexcept -> ExecutionResult {
```

Comment:
```suggestion
                                  int depth) noexcept {
```
You don't need to specify the return type in many places.